packages/app/templates/ui-extensions/projects/checkout_ui/README.md

```diff
@@ -0,0 +1,87 @@
+# Checkout-UI extension README
+
+Checkout UI extensions let app developers build custom functionality that merchants can install at defined points in the checkout flow. [More info](https://shopify.dev/api/checkout-extensions/checkout)
+
+
+## Prerequisites
+---
+Make sure you've completed the following prerequisites to develop a Checkout UI extension:
+
+* You have a [Partner Account](https://www.shopify.com/partners).
+* You created a [Development Store](https://shopify.dev/apps/tools/development-stores) and [enabled the developer preview](https://shopify.dev/api/release-notes/developer-previews#previewing-new-features) for checkout UI extensions.
+* Your store is populated with [test products](https://shopify.dev/apps/getting-started/create#step-4-add-and-publish-products-to-your-development-store-for-testing).
+* You have an [ngrok account](https://ngrok.com/) and [auth token](https://dashboard.ngrok.com/auth/your-authtoken).
+    * This is used to preview apps that contain UI extensions with Shopify CLI. More info inthe [ngrok’s documentation](https://ngrok.com/docs#config).
+* You've created a Shopify app that uses [Shopify CLI 3.0 or higher](https://shopify.dev/apps/getting-started/create) (or you've [migrated your existing app](https://shopify.dev/apps/tools/cli/migrate)).
```

Comment:
I think it's important to keep the prerequisites explicit until there are no more options. If the case is so that this extension can only be scaffolded using CLI3 then I'd rather remove